### PR TITLE
Added example for private service connect for google APIs

### PR DESCRIPTION
**Pull request by betsy-lichtenberg**
Planning to use here:

https://cloud.devsite.corp.google.com/vpc/docs/configure-private-service-connect-apis

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: examples/private_service_connect_google_apis/variables.tf [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "project" {
+  description = "The Google Cloud project ID"
+}
+
+
```
Comment (bharathkkb) [inline examples/private_service_connect_google_apis/variables.tf:21]: nitL newlines

File: examples/private_service_connect_google_apis/main.tf [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+ * limitations under the License.
+ */
+
+
+terraform {
+  required_providers {
+    google = {
+      version = ">= 3.45.0"
+    }
+    null = {
```
Comment (bharathkkb) [inline examples/private_service_connect_google_apis/main.tf:23]: We don't seem to be using this provider in config?

File: examples/private_service_connect_google_apis/main.tf [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+terraform {
+  required_providers {
+    google = {
```
Comment (bharathkkb) [inline examples/private_service_connect_google_apis/main.tf:20]: The configs below reference `google-beta` provider

**Review by bharathkkb — APPROVED** [review]
LGTM
minor nits

File: examples/private_service_connect_google_apis/main.tf (hunk 2 above)
Comment (betsy-lichtenberg) [reply examples/private_service_connect_google_apis/main.tf:23]: I confirmed that this example doesn't work without google-beta, so I kept that and removed the other provider block.

File: examples/private_service_connect_google_apis/variables.tf (hunk 1 above)
Comment (betsy-lichtenberg) [reply examples/private_service_connect_google_apis/variables.tf]: fixed

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**